packages/mandelbrot/assets/scss/components/_tree.scss

```diff
@@ -142,3 +142,8 @@
     padding-top: 0.375rem;
     opacity: 0.7;
 }
+
+.Tree-asideTerm,
+.Tree-asideDescription {
+    display: inline;
+}
```

Comment:
It happens already with the current code but wondering if we allow more and possibly longer values, maybe we shouldn’t wrap it on multiple line 🤔 

![Capture d’écran 2020-10-25 à 19 44 00](https://user-images.githubusercontent.com/793344/97115974-8beafb80-16fa-11eb-8630-2e4ba85523d6.png)

Reply:
Fair, but the current CSS rule affects the entire navigation sidebar, so I'm reluctant to change this here - I think this kind of change should go in the changelog as a separate entry, so maybe a separate issue/pr would be better?

Reply:
Agree